test/kb_ea_utils_server_test.py

```diff
@@ -473,7 +554,9 @@ def getPairedEndLib_SetInfo(self, read_libs_basename_list, refresh=False):
 
     ### TESTS 0.1-0.4: get Fastq Stats
     #
-    def test_get_fastq_ea_utils_stats(self):
+    @patch('biokbase.workspace.client.Workspace.get_objects',
+           side_effect=AssertionError('\n\n*****  Replace get_objects() with get_objects2()'))
+    def test_get_fastq_ea_utils_stats(self, mock):
```

Comment:
I don't think we communicated well regarding the GO1 vs 2 change. Here's my thought process:

- GO1 has been changed to GO2
- presumably this means that some feature of GO2 is being used that does not exist in GO1
- There were no test changes. That means the usage of this new feature is untested
- Therefore, a test(s) should be added so that said usage is tested.
- That test would be expected to fail if someone changed GO2 back to GO1.

I wasn't looking for a mock reaching into the implementation to test that GO1 isn't being used, I was looking for a test that exercises currently untested functionality.